lib/salus/scanners/gitleaks.rb

```diff
@@ -0,0 +1,109 @@
+require "salus/scanners/base"
+require "json"
+
+# Gitleaks scanner integration. Detects hardcoded secrets like passwords,
+# api keys, and tokens in git repos.
+# https://github.com/zricethezav/gitleaks
+
+module Salus::Scanners
+  class Gitleaks < Base
+    def run
+      Dir.chdir(@repository.path_to_repo) do
+        copts = config_options
+        shell_return = run_shell(
+          'gitleaks '\
+          '--path . '\
+          '--format sarif '\
+          '--report /dev/stdout '\
+          '--leaks-exit-code 0 ' +
+          copts
+        )
+
+        report_stderr(shell_return.stderr)
+
+        # A non-zero exit code is definitely an error
+        if !shell_return.success?
+          report_stdout(shell_return.stdout)
```

Comment:
can you change `report_stdout` to `report_stderr`?

Reply:
Should you pass the stuff from both `shell_return.stdout` and `shell_return.stderr` to `report_stderr`?